src/storage/txn/actions/tests.rs

```diff
@@ -417,9 +427,16 @@ pub fn must_rollback<E: Engine>(engine: &E, key: &[u8], start_ts: impl Into<Time
     let snapshot = engine.snapshot(Default::default()).unwrap();
     let start_ts = start_ts.into();
     let cm = ConcurrencyManager::new(start_ts);
-    let mut txn = MvccTxn::new(snapshot, start_ts, true, cm);
-    txn.collapse_rollback(false);
-    txn::cleanup(&mut txn, Key::from_raw(key), TimeStamp::zero(), false).unwrap();
```

Comment:
Now `must_rollback` and `must_rollback_collapsed` are identical, we can keep just one of them.
However some tests are based on non-`collapse_rollback` version of this function, I'm trying to do something.